data/Sri Guru Granth Sahib Ji/0199.json

```
@@ -355,7 +355,7 @@
         "translations": {
           "English": {
             "Dr. Sant Singh Khalsa": {
-              "translation": "Their works are brought to fruition,",
+              "translation": "Their works are brought to fruition",
```

A little tougher since it's a hyphen and it's reversed, but I would suggest the following:

```suggestion
              "translation": "Their works are brought to fruition -",
```